test/parallel/test-http2-server-shutdown-redundant.js

```diff
@@ -0,0 +1,28 @@
+// Flags: --expose-http2
+'use strict';
+
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+const assert = require('assert');
+const http2 = require('http2');
+
+const server = http2.createServer();
+
+// Test blank return when a stream.session.shutdown is called twice
+server.on('stream', common.mustCall((stream) => {
+  stream.session.shutdown(common.mustCall(() => {
+    assert.strictEqual(
+      stream.session.shutdown(),
+      undefined
```

Comment:
I think it's better if this also checks that it `doesNotThrow`.

Reply:
`assert.strictEqual` will ensure that no error is thrown
Tested on [repl](https://repl.it/LaoE/1)